### ISSUE TITLE
Defect - Hide Features not implemented - minor issue

### ISSUE THREAD

**Issue by karengreene** [midohiofoodbank/freshtrak-public#67]
Please hide the following links/buttons as the functionality is not yet implemented:

At the top of the page:
- SignIn
- For Food Banks

Middle of Page
- Create An Account
- Register Food Bank

Bottom of Page
- Register with FreshTrak
- Register Your Foodbank

**Comment by davidpickering** [comment]
Yep, the Footer looks great, but we just don't have that content ready yet.

![no active destination - remove the link](https://user-images.githubusercontent.com/11682225/79936609-e80c5480-8425-11ea-8ee7-378cafa547fd.png)
